tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AspNet/AspNetMvc5Tests.cs

```diff
@@ -173,6 +174,110 @@ public AspNetMvc5TestsInferredProxySpansDisabled(IisFixture iisFixture, ITestOut
         }
     }
 
+    [UsesVerify]
+    public abstract class AspNetMvc5TestsWithBaggage : TracingIntegrationTest, IClassFixture<IisFixture>, IAsyncLifetime
+    {
+        private readonly IisFixture _iisFixture;
+        private readonly string _testName;
+        private readonly bool _classicMode;
+        private readonly bool _enableInferredProxySpans;
+
+        protected AspNetMvc5TestsWithBaggage(IisFixture iisFixture, ITestOutputHelper output)
+            : base("AspNetMvc5", @"test\test-applications\aspnet", output)
+        {
+            SetServiceVersion("1.0.0");
+            SetEnvironmentVariable(ConfigurationKeys.FeatureFlags.RouteTemplateResourceNamesEnabled, true.ToString());
+            SetEnvironmentVariable(ConfigurationKeys.ExpandRouteTemplatesEnabled, false.ToString());
+            SetEnvironmentVariable(ConfigurationKeys.FeatureFlags.TraceId128BitGenerationEnabled, false.ToString());
+            SetEnvironmentVariable(ConfigurationKeys.FeatureFlags.InferredProxySpansEnabled, false.ToString());
+
+            _classicMode = false;
+            _enableInferredProxySpans = false;
+            _iisFixture = iisFixture;
+            _iisFixture.ShutdownPath = "/home/shutdown";
+
+            _testName = $"{nameof(AspNetMvc5Tests)}.WithBaggage";
```

Comment:
To make it easier for me to review the snapshots, can you update the _testName so that it has the same naming convention as the other tests, following this logic?

https://github.com/DataDog/dd-trace-dotnet/blob/98f5d1c35d36edb53b6bc477916788592354c152/tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AspNet/AspNetMvc5Tests.cs#L211-L217

I'm happy to update the snapshots after you make the change